### PR TITLE
Tango Status Page

### PR DESCRIPTION
**Pull request by ymzong**
Implementation of Tango Status page.

Final version of the dashboard is as follows -- it's ready to be merged!

autolab1

### CODE REVIEW

**Comment by ymzong** [comment]
@mihirpandya ^

**Comment by icanberk** [comment]
@ymzong especially for non-admin instructors, I think the list of jobs are more important. Can we move "Currently Running jobs" and "Jobs Waiting to Run" to the top of the page?

**Comment by ymzong** [comment]
Sure, I can definitely pull the modules to top-of-page.

**Comment by ymzong** [comment]
Hey @icanberk --

The PR (#482) from my own working branch to here fixes the issue you mentioned.

Also, if you take a look at #482, you will see that it implements a time-series chart of job lengths, which can be filtered by pool names. Feel free to check it out on staging! :smile:

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -157,6 +157,26 @@ def getjob
...
   end
 
+  action_auth_level :tango_status, :instructor
+  def tango_status
+    # Obtain overall Tango info and pool status
+    @tango_info = TangoClient.info
+    @vm_pool_list = TangoClient.pool
+    # Obtain Image -> Course mapping
+    @img_to_course = { }
+    Assessment.all.each do |asmt|
```
Comment (dlbucci) [inline app/controllers/jobs_controller.rb]: This should be a find_each.

File: app/views/jobs/tango_status.html.erb [hunk 2]
```diff
@@ -0,0 +1,277 @@
+<% @title = "Tango Status" %>
```
Comment (dlbucci) [inline app/views/jobs/tango_status.html.erb]: Could we clean up the tabulation in this file?  2 spaces everywhere?  This looks like a weird mix of different space amounts and hard tabs.

**Comment by dlbucci** [comment]
Make sure you run rubocop on this.  Also, could we update the homes#vmlist route? Either get it to share code with this page, or even just remove it and fix the link, if it doesn't seem to serve any purpose any more.

**Comment by ymzong** [comment]
@dlbucci: All of your concerns are addressed:
- vmlist is removed, and the link is redirected to TangoStatus page;
- I ran rubocup on the code, and also beautified the erb.

**Comment by dlbucci** [comment]
When you say beautify, are you doing something automated to clean it up?

**Comment by ymzong** [comment]
Yep I ran an automated routine, verified that no ERB has been messed with, and tested to make sure it works :smile_cat:

**Comment by dlbucci** [comment]
What automated routine?  Does rubocop work on erb files?

**Comment by ymzong** [comment]
No no, I used an online formatter.

**Comment by ymzong** [comment]
I just did some code clean-up and UI enhancement, and I'm very happy about the code now :smile_cat:

**Comment by icanberk** [comment]
:+1:

**Pull request merged**